src/auth/performance.service.ts

```
@@ -0,0 +1,77 @@
+      if (job.createdAt) {
+        const month = job.createdAt.toISOString().slice(0, 7); // YYYY-MM
+        earningsByMonth[month] = (earningsByMonth[month] || 0) + (job.budget || 0);
+      }
+    });
+    Object.keys(earningsByMonth).sort().forEach(month => earningsOverTime.push(earningsByMonth[month]));
+
+    // Skill demand analysis (top skills in jobs applied to)
+    const applications = await this.applicationRepo.find({ where: { freelancerId }, relations: ['job'] });
+    const skillCount: Record<string, number> = {};
+    applications.forEach(app => {
+      const jobSkills = app.job && app.job['skills'];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)